x/vesting/keeper/msg_server_test.go

```diff
@@ -475,3 +475,74 @@ func (suite *KeeperTestSuite) TestClawbackVestingAccountMarshal() {
 	_, err = suite.app.AccountKeeper.UnmarshalAccount(bz[:len(bz)/2])
 	suite.Require().Error(err)
 }
+
+func (suite *KeeperTestSuite) TestConvertVestingAccount() {
+	testCases := []struct {
+		name     string
+		malleate func() authtypes.AccountI
+		expPass  bool
+	}{
+		{
+			"fail - not a account found",
+			func() authtypes.AccountI {
+				from, priv := testutil.NewAccAddressAndKey()
+				baseAcc := authtypes.NewBaseAccount(from, priv.PubKey(), 1, 5)
+				return baseAcc
+			},
+			false,
+		},
+		{
+			"fail - not a vesting account",
+			func() authtypes.AccountI {
+				from, priv := testutil.NewAccAddressAndKey()
+				baseAcc := authtypes.NewBaseAccount(from, priv.PubKey(), 1, 5)
+				suite.app.AccountKeeper.SetAccount(suite.ctx, baseAcc)
+				return baseAcc
+			},
+			false,
+		},
+		{
+			"fail - still vesting",
```

Comment:
@Vvaradinov did you test when the account has locked tokens?

Reply:
@fedekunze Yes, when creating a new ClawbackVestingAccount I provide both a lockup file and a vesting file so it should cover that case.

Reply:
Actually now that I'm looking at it, I think the `GetVestingCoins` should be enough to check since it will return `nil` if the tokens are locked.

Reply:
@Vvaradinov let's add the following tests cases:

-  `success - unlocked & vested`
- `fail - unlocked & unvested`
- `fail - locked & vested`
- `fail - locked & unvested`